mixlr-repeater.gemspec

```diff
@@ -1,6 +1,8 @@
+require_relative 'lib/version'
```

Comment:
Yes you're right. Or I think we could use `require_relative` and add `s.require_paths = ['lib']` to the gemspec. Will fix up.